bus-mapping/src/circuit_input_builder/execution.rs

```diff
@@ -1040,6 +1040,15 @@ pub struct EcMulOp {
     pub r: Option<G1Affine>,
 }
 
+/// Constant representing the modulus
+/// r = 0x30644e72e131a029b85045b68181585d2833e84879b9709143e1f593f0000001
+pub const BN256_FR_MODULUS: Fr = Fr::from_raw([
```

Comment:
this is essentially `Fr::zero()`. although it has a same internal representation as `modulus`, it is quite ambiguous to define the modulus this way IMHO.


I think it is cleaner is to define 
```
/// Constant representing the modulus - 1
/// r - 1 = 0x30644e72e131a029b85045b68181585d2833e84879b9709143e1f593f0000000
pub const BN256_FR_MODULUS: Fr = Fr::from_raw([
    0x43e1f593f0000000,
    0x2833e84879b97091,
    0xb85045b68181585d,
    0x30644e72e131a029,
]);
```

since we only uses `-1` rather than `modulus` in line 1116